specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/mfe.json

```diff
@@ -13277,6 +13330,27 @@
       },
       "additionalProperties": false
     },
+    "CapacityReservation": {
+      "required": [
+        "capacityReservationGroupId"
+      ],
+      "type": "object",
+      "properties": {
+        "capacityReservationGroupId": {
```

Comment:
>capacityReservationGroupId [](http://example.com/codeflow?start=9&length=26)

Pattern makes this look more like a name. Why is it called an ID?

Reply:
It will be the ARM resource ID of the capacity reservation resource.